### ISSUE TITLE
pre-trained models

### ISSUE THREAD

**Issue by aashusri** [facebookresearch/brainmagick#2]
Thanks for the great work!

Are you planning to make the pre-trained models public?

**Comment by kingjr** [comment]
Unfortunately no, we will not release the pretrained models.

**Comment by vocaliodmiku** [comment]
@aashusri Hi, I followed the README and I have reproduced the model on 2xA40 GPU (~6hours) with dataset ['gwilliams2022']. I eval the checkpoint on the test set and get a TOP-10 acc 70.608. if you need it, please access this [link](https://drive.google.com/drive/folders/1MP8JjgF01OaEcQHCb7_5qUkkvtKBa1K1?usp=sharing).

**Comment by elisim** [comment]
@vocaliodmiku how TOP-10 acc 70.608 is differ from what reported in the paper?

**Comment by vocaliodmiku** [comment]
> @vocaliodmiku how TOP-10 acc 70.608 is differ from what reported in the paper?

I think it matches what reported in the paper (70.7±0.1).

**Comment by NeuSpeech** [comment]
> @vocaliodmiku how TOP-10 acc 70.608 is differ from what reported in the paper?

Hey, is it possible for you to share pretrained models?
